roaringbitmap--0.5--0.6.sql

```diff
@@ -0,0 +1,6 @@
+/* roaringbitmap--0.5--0.6 */
+
+CREATE OR REPLACE FUNCTION rb_runoptimize(roaringbitmap)
```

Comment:
what's the plan? to use this function in updates from time to time?

Reply:
yes. the time series team have a script that, currently, every 5 minutes condenses all the raw bitmap data in to a weekly bucket. Optimizing on conflict should allow these bitmaps to be more compact, when possible.